### ISSUE TITLE
Confusing "New translation" and "zh_Hans"

### ISSUE THREAD

**Issue by stanislav-brabec** [nijel/weblate#1162]
Field titled "New translation" is a bit confusing, and some users mistakenly assume that it means "Join to the translation". It should be probably named "Add new language".

Additionally, it would be useful to mention language code in the drop down menu. It is easy to be confused e. g. by zh_Hans (Chinese in the simplified script) mentioned as "Simplified Chinese" and zh_TW mentioned as "Chinese (Taiwan)", which is often called Simplified Chinese as well.

### Steps to reproduce
1. Go to any subproject.

### Actual behaviour
New translation -> Simplified Chinese
zh_Hans is added

### Expected behaviour
Rename to "Add new language" or so.
Add language codes to drop down menu.
prevent zh_Hans confusion and rename e. g. to "Chinese in the simplified script" (needs check by somebody who knows Chinese).

### Server configuration
 * Weblate 2.6
 * Python 2.7.9
 * Django 1.8.4
 * six 1.9.0
 * python-social-auth 0.2.7
 * Translate Toolkit 1.14.0-rc1
 * Whoosh 2.7.0
 * Git 1.8.5.6
 * Pillow (PIL) 1.1.7
 * dateutil 2.4.2
 * lxml 3.2.3
 * django-crispy-forms 1.5.2
 * compressor 1.5
 * djangorestframework 3.3.3
 * pytz 2013.9
 * pyuca N/A
 * Database backends: django.db.backends.mysql

**Comment by wsxy162** [comment]
zh_TW is traditional Chinese.

**Comment by marguerite** [comment]
Here's the thing.

Literally, zh_Hans/zh_Hant is more widely covered than zh_CN/zh_TW.

Because Singapore use Simplified Chinese, the same version as Mainland China, too. If someone set locale to zh_SG.UTF-8, the zh_CN translations will not apply, but zh_Hans might be. The same thing applies to HongKong (zh_HK), Macau (zh_blabla), which will load zh_Hant, but not zh_TW.

That's just in theory. 

In reality, people just take zh_CN/zh_TW and zh_Hans/zh_Hant as two sets of standards. and the zh_Hans/zh_Hant standard is rarely used (mostly if the project is set up by a foreign guy, he may choose zh_Hans according to wikipedia or some English tutorials, mostly no Chinese will choose that). 

And those regions (HongKong, Macau, Singapore) has few open source population so that they may not afford to maintain a separate set of translation. Usually they use zh_CN.UTF-8 locale, like this. And as their official working languages include English, people may use English too.

Basically, maintaining zh_Hans/zh_CN at the same time is useless (Sometimes there're some dialects/variants, but as the characters are familiar to them, it's not that hard to understand and it is a cost that can bear).

I think we can add some functions to warn people who want to create a zh_Hans/zh_Hant language when zh_CN/zh_TW exists. Because basically it's a waste of time and it is most likely that people hit the wrong button. And link zh_Hans with zh_CN if there's no separate zh_Hans exists.

**Comment by stanislav-brabec** [comment]
wsxy162: Thanks for explanation.

The warning for some combinations of languages seems to be reasonable:

Also a warning while creating country specific translations if a country unspecific translations does not exist makes sense. (e. g. warn while creating "es_AR" but "es" does not exist). Exceptions from this rule could be: zh_* (no zh exist) pt_BR (too different from pt to be useful to share them).

Tools for tracking of similar language don't exist in weblate yet (e. g. track all changes in zh_Hans as a special type of suggestions for zh_TW and vice versa).

Weblate now only features so called "Secondary languages" (in user preferences). Maybe it would be interesting to add "Take" button at the displayed string in the secondary language, so the translator can take the string from a similar language without a change.

**Comment by nijel** [comment]
I've created separate issue for the languages validation, feel free to comment there on this topic: https://github.com/nijel/weblate/issues/1172
